.github/workflows/Python.yml

```diff
@@ -432,14 +432,15 @@ jobs:
           ./scripts/upload-assets-to-staging.sh twine_upload tools/pythonpkg/wheelhouse/*.whl
 
   linux-release-cleanup:
-     if: startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/main'
+     if: github.ref == 'refs/heads/v1.2-histrionicus' || github.ref == 'refs/heads/main'
      name: PyPi Release Cleanup
      needs: twine-upload
      runs-on: ubuntu-22.04
      env:
        PYPI_CLEANUP_USERNAME: 'mytherin'
        PYPI_CLEANUP_PASSWORD: ${{secrets.PYPI_CLEANUP_PASSWORD}}
        PYPI_CLEANUP_OTP: ${{secrets.PYPI_CLEANUP_OTP}}
+       DUCKDB_BASE_VERSION: ${{github.ref == 'refs/heads/main' && '1.3' || '1.2'}}
```

Comment:
This part is the weirder one, but I am not sure if / what is a more proper alternative. This would need to be revised / changed moving forward. I don't have a proper answer, possibly it's fine and to be generalised later (together with removing the hardcoded branch name), first checking this work as intended.